projects/components/src/sharing-modal/select-all-checkbox/select-all-toggle.component.ts

```diff
@@ -0,0 +1,67 @@
+/*!
+ * Copyright 2021 VMware, Inc.
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+import { Component, EventEmitter, Input, Output } from '@angular/core';
+import { LazyString } from '@vcd/i18n';
+
+export interface ComboOption {
+    display: LazyString;
+    value: string;
+}
+
+/**
+ * A togggle that allows you to select all and choose the type of right to apply.
+ */
+@Component({
+    selector: 'vcd-select-all-toggle',
+    templateUrl: 'select-all-toggle.component.html',
+    styleUrls: ['./select-all-toggle.component.scss'],
+})
+export class VcdSelectAllToggleComponent {
```

Comment:
Shouldn't this be a control value accessor like the rest of our form controls?

Reply:
We talked and decided that this is fine left as not a form control because it is not a reusable component.